src/OpenLoco/src/Ui/Windows/ToolbarTopCommon.cpp

```diff
@@ -44,6 +44,7 @@ namespace OpenLoco::Ui::Windows::ToolbarTop::Common
         const auto companyColour = CompanyManager::getPlayerCompanyColour();
 
         auto lastRoadOption = getGameState().lastRoadOption;
+        auto lastRailRoadOption = getGameState().lastRailroadOption;
```

Comment:
Oddly enough this is not what vanilla is doing at this point. Only `lastRoadOption` is used.